src/components/pages/Fuse/FusePoolPage.tsx

```diff
@@ -155,7 +162,7 @@ const CollateralRatioBar = ({
         </SimpleTooltip>
 
         <Text flexShrink={0} mt="2px" mr={3} fontSize="10px">
-          0%
+          {ratio.toFixed(1)}%
```

Comment:
I don't like displaying the ratio here either - forgot to fix - but I think showing $USD value of your borrow amount might reinforce the user's understanding of their position - something to toy with, can change it back to `0%` if none work

Reply:
Nah I think it's cool, agreed that it would be nice to show that stuff. just maybe a different spot? idk have to think on it a little more. feel free to leave it in